src/poliastro/twobody/orbit.py

```diff
@@ -1509,3 +1509,18 @@ def plot(self, label=None, use_3d=False, interactive=False):
             from poliastro.plotting.core import OrbitPlotter2D
 
             return OrbitPlotter2D().plot(self, label=label)
+
+
+# will be required in future
```

Comment:
If you are planing to implement this some time later, then let us dismiss it from now.

Reply:
yes for sure it has nothing to do with animation, it was planned to be an interface like a thing similar to the above plot function which can direct to corresponding 3d or 2d plot as per user input